src/patternfly/components/Table/table.scss

```diff
@@ -42,8 +42,8 @@
   --pf-c-table-thead-cell--PaddingBottom: var(--pf-global--spacer--md);
 
   // Td
-  --pf-c-table-tbody-cell--PaddingTop: var(--pf-global--spacer--xl);
-  --pf-c-table-tbody-cell--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-table-tbody-cell--PaddingTop: var(--pf-global--spacer--lg);
```

Comment:
You'll have to update the padding when the table breaks into its grid, but before the mobile view: this is currently still 32px.
<img width="692" alt="Screen Shot 2019-04-25 at 5 27 06 PM" src="https://user-images.githubusercontent.com/20118816/56769515-6a185d80-677f-11e9-868f-f74e4c7fc867.png">